src/lib.rs

```diff
@@ -1 +1,12 @@
+//! # Utreexo
+//! [Utreexo](https://eprint.iacr.org/2019/611.pdf) is a novel accumulator that allows for succinct
+//! UTXO set representation, using a logarithmic amount of space. It uses a dynamic accumulator
+//! that allows for the addition and deletion of elements. While spending  a UTXO, the element
+//! is deleted from the accumulator. While receiving a UTXO, the element is added to the accumulator.
+//! During validation, nodes receive a proof that the UTXO they are spending is in the the current UTXO set
```

Comment:
`s/nodes receive a proof that the UTXO they are spending is in the the current UTXO set and has a given value./nodes receive a proof which proves the inputs in a block or transaction exist in the UTXO set./`